packages/client/utils/AtlassianManager.ts

```diff
@@ -398,10 +489,11 @@ export default abstract class AtlassianManager {
     }
     return new Error(`Unknown Jira error: ${JSON.stringify(error)}`)
   }
-  private readonly delete = async (url) => {
+  private readonly delete = async (url, payload?: any) => {
```

Comment:
+1 would be nice to add a type to `url: string,` as well

Reply:
agreed